src/Gui/3Dconnexion/navlib/NavlibNavigation.cpp

```diff
@@ -385,12 +385,12 @@ long NavlibInterface::SetViewExtents(const navlib::box_t& extents)
     return navlib::make_result_code(navlib::navlib_errc::no_data_available);
 }
 
-long NavlibInterface::GetViewFOV(double& fov) const
+long NavlibInterface::GetViewFOV(double&) const
```

Comment:
I swear I saw a lint warning yesterday about missing parameter name. Did we change linter settings?

Reply:
If it was missing in the header file I suspect that would trip such a warning. I could switch to `[[maybe_unused]]` here in the cpp file if you prefer.